src/Rules/StarterWeb/AI/IndividualAuth/appsettings.json

```diff
@@ -2,11 +2,7 @@
   "ApplicationInsights": {
     "InstrumentationKey": ""
   },
-  "Data": {
-    "DefaultConnection": {
-      "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true"
-    }
-  },
+  "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true",
```

Comment:
Also, agreed with @shirhatti on the verbosity of the connection string.It isn't pretty to have to type all that - making any assumptions about the nomenclature users "should use" for their connection string is kind of a bad proposition. I'm also chatting with the App Service folks to make sure there aren't any assumptions made on the names of the connection strings. We should make as few assumptions as possible about prefixes and so on.